exercises/pangram/pangram_test.go

```diff
@@ -0,0 +1,35 @@
+package pangram
+
+import (
+	"testing"
+)
+
+const targetTestVersion = 1
+
+type testCase struct {
+	input    string
+	expected bool
+}
+
+var stringTestCases = []testCase{
+	{"quick movement of the enemy will jeopardize five gunboats", false},
+	{"the quick brown fish jumps over the lazy dog", false},
+	{"7h3 qu1ck brown fox jumps ov3r 7h3 lazy dog", false},
+	{"the 1 quick brown fox jumps over the 2 lazy dogs", true},
+	{"The quick brown fox jumps over the lazy dog", true},
+}
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Errorf("Found testVersion = %v, want %v.", testVersion, targetTestVersion)
+	}
+}
+
+func TestPangram(t *testing.T) {
+	for _, test := range stringTestCases {
+		actual := is_pangram(test.input)
+		if actual != test.expected {
+			t.Errorf("Pangram test [%s], expected [%s], actual [%s]", test.input, test.expected, actual)
```

Comment:
One of the things we try to do is help people understand _why_ a test fails. How about adding a _description_ field to the test case struct that explains the context of the failure (e.g. "missing character $CHAR")